docs/contributing.rst

```diff
@@ -56,6 +67,55 @@ only ask that you are aware of the following:
 
 .. include:: ../CODE_OF_CONDUCT.rst
 
+
+.. _how-to-run-tests:
+
+How To Run The Tests
+--------------------
+
```

Comment:
An explanation for why there are two sets of tests - perhaps refer to them as categories and even add guidance for what tests should be added where in the additional test writing docs?